Dockerfile

```diff
@@ -85,6 +85,13 @@ RUN mkdir -p /usr/local/bin && \
     cd /tmp && \
     rm -rf /tmp/build
 
+# Install prefixout from Trevor
+RUN set -exv \
+  && apk add go \
+  && go get -u -x -v github.com/akatrevorjay/prefixout
```

Comment:
Wouldn't we fork that to a whistle repo?

Reply:
sure

Reply:
updated